target/ppc/translate/vmx-impl.c.inc

```diff
@@ -1352,6 +1352,36 @@ TRANS(VINSHRX, do_vinsx, 2, true, gen_helper_VINSHX)
 TRANS(VINSWRX, do_vinsx, 4, true, gen_helper_VINSWX)
 TRANS(VINSDRX, do_vinsx, 8, true, gen_helper_VINSDX)
 
+static bool trans_VINSW(DisasContext *ctx, arg_VX_uim4 *a)
```

Comment:
Only 2 functions, so no big deal. But we could also use a macro here to generate trans_VINS{W,D}